meta-lmp-base/recipes-core/initrdscripts/initramfs-framework/ostree_factory_reset

```diff
@@ -34,15 +36,34 @@ factory_reset() {
 	if [ "${keep_sota}" = "true" ]; then
 		msg "Keeping current ${OSTREE_DISTRO} SOTA content"
 		FIND_EXCLUDE="${FIND_EXCLUDE} ! -name 'sota'"
+		keep_rpmb="true"
 	fi
 	if [ "${keep_docker}" = "true" ]; then
 		msg "Keeping current docker content"
 		FIND_EXCLUDE="${FIND_EXCLUDE} ! -name 'lib'"
 	fi
 
+	cd ${OSTREE_VAR}
+
+	# Validate if aktualizr-lite was configured to use
+	# RPMB storage for the device certificate and keys.
+	sota_rpmb=$(grep pkey_source sota/sota.toml | cut -d'"' -f 2)
+	if [ "${sota_rpmb}" = "pkcs11" ]; then
+		aktualizr_uses_rpmb="true"
+	fi
+
+	# Cleanup RPMB before /var gets removed since we need some
+	# parameters which are stored in /var/sota/sota.toml
+	if [ "${keep_rpmb}" = "false" ] && [ "${aktualizr_uses_rpmb}" = "true" ]; then
+		ptool_module=$(grep module sota/sota.toml | cut -d'"' -f 2)
+
+		# Reinitializing a PKCS11 token destroys all objects which
+		# were associated with that slot.
+		pkcs11-tool --module ${ptool_module} --init-token --label "unused" --slot ${AKTUALIZR_SLOT}
```

Comment:
Here I assume you are using the pkcs11 OP-TEE implementation (due RPMB access), so this call will only really work in a generic way with the right module, as we could have other pkcs11 providers that are not necessarily provided by OP-TEE (e.g. softhsm2, pkcs11-tpm2, etc).

Since we can assume we're using pkcs11 OP-TEE we can search if the module is libckteec, and skip otherwise.

You might also need the sopin to initialize, but needs testing (it is also available in sota.toml if needed).

One other thing, since the pkcs11 OP-TEE implementation requires that the pkcs11 trusted application to be loaded and operational in optee, you will need to make sure tee-supplicant is running before executing this command (libckteec will request the TA to be loaded via tee-supplicant on first access). You can use https://github.com/foundriesio/meta-lmp/blob/main/meta-lmp-base/recipes-core/initrdscripts/initramfs-framework/cryptfs_pkcs11 as a guide, since it is also executed as part of the initrd.